albinos-editor/src/albinospkg/cli.nim

```diff
@@ -2,24 +2,55 @@ import terminal
 import strutils
 import rdstdin
 import linenoise
+import cmd
+import ../libalbinos/albinos
 
-let doc = """
-Usage:
-    help (clear the screen)
-    exit (quit the cli)
-    clear (clear the terminal)
-    config create <name> (create a config with the given name)
-    config load <key> (load a config with the given key)
+const help_cmd_msg = "\t\e[93mhelp\e[39m (show this message)\n\t"
+const exit_cmd_msg = "\e[93mexit\e[39m (quitting the app)\n\t"
+const clear_cmd_msg = "\e[93mclear\e[39m (clear the screen)\n\t"
+const config_create_cmd_msg = "\e[93mconfig\e[39m \e[94mcreate\e[39m <\e[95mname\e[39m> (create a config witht the given name)"
+const global_help = "Usage:\n" & help_cmd_msg & exit_cmd_msg & clear_cmd_msg &
+    config_create_cmd_msg
 
-Example:
-    > config create my_cfg
-"""
 
-proc handleConfigCmd(argsSet: openArray[string]) =
-    echo argsSet
 
-proc launchCLI*() =
-    discard historySetMaxLen(500)
+proc yes(question: string): bool =
+    echo question, " (\e[93my\e[39m/\e[93mn\e[39m)"
+    while true:
+        var line: TaintedString
+        let res = readLineFromStdin("Answer: ", line)
+        if res == false:
+            styledEcho(fgMagenta, "Question Aborted, no is assumed.")
+            return false
+        case line:
+            of "y", "Y", "yes", "Yes": return true
+            of "n", "N", "no", "No": return false
+            else: echo "Please be clear: yes or no"
```

Comment:
Maybe print the received string?
This can help users that may have a keyboard modifier active and wrote "yES" or situations like that.